server/channels/app/platform/service.go

```diff
@@ -247,6 +248,19 @@ func New(sc ServiceConfig, options ...Option) (*PlatformService, error) {
 		ps.filestore = backend
 	}
 
+	if ps.exportFilestore == nil {
+		ps.exportFilestore = ps.filestore
+		if *ps.Config().FileSettings.DedicatedExportStore {
+			insecure := ps.Config().ServiceSettings.EnableInsecureOutgoingConnections
```

Comment:
The insecure part allows server admins to have a minio instance with a self signed certificate. This is the same code we use for the normal storage. Any reason why we would not want to allow this for the backup storage?

Reply:
I think we should aim for the best practices right? Self-signed means we have a certificate without the support of a trusted certificate authority to guarantee the validity of the certificate details.

Reply:
I honestly don't have a strong opinion on that, I figured we would like to have feature parity between the normal filestore and the filestore dedicated to export.
If it's not the case, I don't mind `insecure` to false - trying to get this in ASAP because the deadline is coming!

Reply:
I'd prefer setting `insecure` to false. Feature parity is great only when we still believe that we took the best decisions and followed the best practices when we developed the initial one. Otherwise, let's keep improving :)

Reply:
Agree with @esarafianou

Reply:
Changed! Thanks!